src/styles/global.css

```diff
@@ -102,3 +102,15 @@ h5,
 h6 {
   font-family: Mulish-bold;
 }
+
+@media (max-width: 991px) {
+  .dropleft .dropdown-menu {
+    right: unset !important;
+    left: 3.5em !important;
+    padding: 1.5rem;
+  }
+}
+
+#basic-navbar-nav {
+  padding: 2rem;
+}
```

Comment:
@Aman-Codes you should keep this class inside media query.

![Screenshot from 2021-07-07 19-38-13](https://user-images.githubusercontent.com/56133783/124773935-042ce080-df5b-11eb-88d5-d4b75c21364e.png)